interface/patient_tracker/patient_tracker.php

```diff
@@ -85,553 +85,1174 @@
+    <link rel="stylesheet"
+          href="<?php echo $GLOBALS['assets_static_relative'] ?>/font-awesome-4-6-3/css/font-awesome.min.css">
+    <link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'] ?>/pure-0-5-0/pure-min.css">
+    <link rel="stylesheet"
+          href="<?php echo $GLOBALS['web_root']; ?>/library/css/bootstrap_navbar.css?v=<?php echo $v_js_includes; ?>"
+          type="text/css">
+    <link rel="stylesheet" href="<?php echo $css_header; ?>" type="text/css">
+    <link rel="stylesheet"
+          href="<?php echo $GLOBALS['assets_static_relative'] ?>/jquery-ui-1-12-1/themes/cupertino/jquery-ui.css" />
 
-// used to display the patient demographic and encounter screens
-function topatient(newpid, enc) {
- if (document.pattrk.form_new_window.checked) {
-   openNewTopWindow(newpid,enc);
- }
- else {
-   top.restoreSession();
-   if (enc > 0) {
-     top.RTop.location= "<?php echo $GLOBALS['webroot']; ?>/interface/patient_file/summary/demographics.php?set_pid=" + newpid + "&set_encounterid=" + enc;
-   }
-   else {
-     top.RTop.location = "<?php echo $GLOBALS['webroot']; ?>/interface/patient_file/summary/demographics.php?set_pid=" + newpid;
-   }
- }
-}
+    <link rel="shortcut icon" href="<?php echo $webroot; ?>/sites/default/favicon.ico" />
 
-// opens the demographic and encounter screens in a new window
-function openNewTopWindow(newpid,newencounterid) {
- document.fnew.patientID.value = newpid;
- document.fnew.encounterID.value = newencounterid;
- top.restoreSession();
- document.fnew.submit();
- }
+    <script type="text/javascript"
+            src="<?php echo $GLOBALS['web_root']; ?>/library/dialog.js?v=<?php echo $v_js_includes; ?>"></script>
+    <script type="text/javascript"
+            src="<?php echo $GLOBALS['web_root']; ?>/library/textformat.js?v=<?php echo $v_js_includes; ?>"></script>
+    <script src="<?php echo $GLOBALS['assets_static_relative'] ?>/jquery-min-3-1-1/index.js"></script>
+    <script src="<?php echo $GLOBALS['assets_static_relative'] ?>/jquery-ui-1-11-4/jquery-ui.min.js"></script>
+    <script src="<?php echo $GLOBALS['assets_static_relative'] ?>/bootstrap-3-3-4/dist/js/bootstrap.min.js"></script>
+    <script src="<?php echo $GLOBALS['assets_static_relative'] ?>/jquery-migrate/jquery-migrate-3.0.0.min.js"></script>
+    <script type="text/javascript"
+            src="<?php echo $GLOBALS['web_root']; ?>/interface/main/messages/js/reminder_appts.js?v=<?php echo $v_js_includes; ?>">
+    </script>
 
-</script>
+    <script>
+        <?php
+        if ($GLOBALS['date_display_format'] == '0') {
+            $date_format = 'yy-m-d';
+        } elseif ($GLOBALS['date_display_format'] == '1') {
+            $date_format = 'mm/dd/yy';
+        } elseif ($GLOBALS['date_display_format'] == '2') {
+            $date_format = 'dd/mm/yy';
+        }
+        ?>
+        var xljs_dateFormat = '<?php echo $date_format; ?>';
```

Comment:
TODO
(remove this and this code block in the messages script after have fully converted to the standard datepicker)